### DISCUSSION TITLE
Which model is used in the dashboard demo: Transcribe your voice live?

### DISCUSSION THREAD

**Discussion by BarnoldSporrance** [deepgram/community#431] [General help] [answered]
Hi,

I'm new to Deepgram.  I'm just wondering which model is used in the **dashboard demo: Transcribe your voice live**?

Is it the enhanced model?

Thanks in advance!

**Comment by jpvajda** [comment, marked as answer]
@BarnoldSporrance  It really depends on which language you are picking. In The Deepgram Console Mission Demo what you can do is pick your language  in the drop down and speak.  If you open the Browser Inspector and view the network tab you can see the API request being made which includes the model being requested.

You can also refer to our [Language Overview ](https://developers.deepgram.com/docs/languages-overview)documentation for more information on which models support which languages. 



![Screenshot 2023-11-09 at 8 33 56 PM](https://github.com/deepgram/community/assets/1395158/86aa132c-1e32-48ce-87ba-68ad79809717)

**Reply by BarnoldSporrance** [reply]
Great @jpvajda  - thanks very much!
